fastly/waf.go

```diff
@@ -15,11 +14,11 @@ type WAFConfigurationSet struct {
 
 // WAF is the information about a firewall object.
 type WAF struct {
-	ID                string     `jsonapi:"primary,waf"`
-	Version           int        `jsonapi:"attr,version"`
-	PrefetchCondition string     `jsonapi:"attr,prefetch_condition"`
-	Response          string     `jsonapi:"attr,response"`
-	LastPush          *time.Time `jsonapi:"attr,last_push"`
+	ID                string `jsonapi:"primary,waf"`
+	Version           int    `jsonapi:"attr,version"`
+	PrefetchCondition string `jsonapi:"attr,prefetch_condition"`
+	Response          string `jsonapi:"attr,response"`
+	LastPush          string `jsonapi:"attr,last_push"`
```

Comment:
Is there a reason this is a string instead of a time? It appears to be a time value.

Reply:
@sethvargo  Yeah, because of issue #54

Reply:
I could not get this to properly unmarshal into a time value, so I changed it to a string for the purposes of being able to use this while I was making changes. I'm open to feedback here also